unwind-itanium.h

```diff
@@ -33,6 +33,11 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 extern "C" {
 #endif
 
+/* definitions below do not handle LLP64, so use compiler definitions */
+#ifdef __MINGW32__
+#include <unwind.h>
+#else
```

Comment:
Did you test whether this actually works?  I don't know how MinGW handles exceptions - does it use UNIX-style DWARF unwinding, or does it use the Win64 EH ABI?  If it's the latter, then we should get the patches from Microsoft and default to that.

Reply:
Haven't been able to check this change because of the link errors listed above. Any idea how I can work around that?